app/common/state/tabState.js

```diff
@@ -610,7 +610,7 @@ const tabState = {
     state = tabState.removeTabField(state, 'messageBoxDetail')
     state = tabState.removeTabField(state, 'frame')
     state = state.delete('tabsInternal')
-    return state.delete('tabs')
+    return state.set('tabs', Immutable.List())
```

Comment:
that would be perfectly fine to not delete the state.  It's just that if you put undefined for 'tabs' in state here then it won't get filled later.